app.js

```diff
@@ -1,102 +1,111 @@
-  new OddDuckProduct('bathroom', 'img/bathroom.jpg'),
-  new OddDuckProduct('boots', 'img/boots.jpg'),
-  new OddDuckProduct('breakfast', 'img/breakfast.jpg'),
-  new OddDuckProduct('bubblegum', 'img/bubblegum.jpg'),
-  new OddDuckProduct('chair', 'img/chair.jpg'),
-  new OddDuckProduct('cthulhu', 'img/cthulhu.jpg'),
-  new OddDuckProduct('dog-duck', 'img/dog-duck.jpg'),
-  new OddDuckProduct('dragon', 'img/dragon.jpg'),
-  new OddDuckProduct('pen', 'img/pen.jpg'),
-  new OddDuckProduct('pet-sweep', 'img/pet-sweep.jpg'),
-  new OddDuckProduct('r2bag', 'img/r2bag.jpg'),
-  new OddDuckProduct('scissors', 'img/scissors.jpg'),
-  new OddDuckProduct('shark', 'img/shark.jpg'),
-  new OddDuckProduct('tauntaun', 'img/tauntaun.jpg'),
-  new OddDuckProduct('unicorn', 'img/unicorn.jpg'),
-  new OddDuckProduct('water-can', 'img/water-can.jpg'),
-  new OddDuckProduct('wine-glass', 'img/wine-glass.jpg'),
+  new OddDuckProduct("baby-sweep", "img/baby-sweep.jpg"),
+  new OddDuckProduct("banana", "img/banana.jpg"),
+  new OddDuckProduct("bathroom", "img/bathroom.jpg"),
+  new OddDuckProduct("boots", "img/boots.jpg"),
+  new OddDuckProduct("breakfast", "img/breakfast.jpg"),
+  new OddDuckProduct("bubblegum", "img/bubblegum.jpg"),
+  new OddDuckProduct("chair", "img/chair.jpg"),
+  new OddDuckProduct("cthulhu", "img/cthulhu.jpg"),
+  new OddDuckProduct("dog-duck", "img/dog-duck.jpg"),
+  new OddDuckProduct("dragon", "img/dragon.jpg"),
+  new OddDuckProduct("pen", "img/pen.jpg"),
+  new OddDuckProduct("pet-sweep", "img/pet-sweep.jpg"),
+  new OddDuckProduct("r2bag", "img/r2bag.jpg"),
+  new OddDuckProduct("scissors", "img/scissors.jpg"),
+  new OddDuckProduct("shark", "img/shark.jpg"),
+  new OddDuckProduct("tauntaun", "img/tauntaun.jpg"),
+  new OddDuckProduct("unicorn", "img/unicorn.jpg"),
+  new OddDuckProduct("water-can", "img/water-can.jpg"),
+  new OddDuckProduct("wine-glass", "img/wine-glass.jpg"),
 ];
 
 // Render method
 
-OddDuckProduct.prototype.render = function(n){
-
+OddDuckProduct.prototype.render = function (n) {
   let imgContainer = document.getElementById(`image${n}`);
-  imgContainer.src = (this.imgPath);
+  imgContainer.src = this.imgPath;
 };
 
 //Implement Local Storage / Persistent data between refreshes/restarts.
 
-
-function saveInitial(){
-
-  let savedProducts = [];
-
-  for(let i = 0; i < allProducts.length; i++) {
+function save() {
+  for (let i = 0; i < allProducts.length; i++) {
     let product = allProducts[i];
     savedProducts.push(product);
   }
   savedProducts = JSON.stringify(savedProducts);
-  localStorage.setItem('productData', savedProducts);
+  localStorage.setItem("productData", savedProducts);
 }
 
-OddDuckProduct.prototype.loadPriorData = function (){
-
-  let savedProducts = localStorage.getItem('productData');
-
-  savedProducts = JSON.parse(savedProducts);
-  savedProducts.push(this);
+function load() {
+  let parsedProducts = localStorage.getItem("productData");
+  JSON.parse(parsedProducts);
```

Comment:
You need to store the result of your JSON.parse somewhere, with your code as it is now you parse it and then it's gone.